content/whitepaper/protocol/crypto-primitives.mdx

```diff
@@ -72,6 +72,7 @@ This is an overview of all the cryptographic keys and cryptographic schemes used
 | Zero-Knowledge Proofing  | Proofing of spends, outputs, and mints       | Groth16 zk-SNARK on BLS12                          | 126 bits                                                         |
 | Hash Tree                | Merkle Tree of Notes                         | Merkle Tree with Pedersen Hash on the JubJub group | 126 bits (preimage and collision resistance)                     |
 | Hashing                  | Mining                                       | Blake3                                             | 256 bits (preimage resistance) / 128 bits (collision resistance) |
+| Hashing                  | Mining                                       | FishHash                                           | 128 bits                                                         |
```

Comment:
Can we say something like: "Mining (until block 503337)" and "Mining (from block 503338 onwards)"?

This does not properly capture testnet/devnet, but I think it'd make the table less ambiguous